libcloud/test/compute/test_gce.py

```diff
@@ -983,10 +991,12 @@ def test_ex_create_route(self):
     def test_ex_create_network(self):
         network_name = 'lcnetwork'
         cidr = '10.11.0.0/16'
-        network = self.driver.ex_create_network(network_name, cidr)
+        routing_mode = 'REGIONAL'
+        network = self.driver.ex_create_network(network_name, cidr, routing_mode='regional')
```

Comment:
Maybe we should still test the default value too? This would have avoided the bug above.

Reply:
Maybe test both (realized I can also add test coverage for "description" field while I'm at it:
```python
    def test_ex_create_network(self):
        network_name = 'lcnetwork'
        cidr = '10.11.0.0/16'
        description = 'A custom network'
        routing_mode = 'REGIONAL'

        # Test defaults
        network = self.driver.ex_create_network(network_name, cidr)
        self.assertTrue(isinstance(network, GCENetwork))
        self.assertEqual(network.name, network_name)
        self.assertEqual(network.cidr, cidr)

        # Test using more options
        network = self.driver.ex_create_network(network_name, cidr,
                                                description=description,
                                                routing_mode=routing_mode)
        self.assertEqual(network.description, description)
        self.assertEqual(network.extra['routingConfig']['routingMode'], routing_mode)
```

Just saying list of dicts is enough for the subnetwork docstring, I guess.

Reply:
Sounds good!